lib/reporting/drop_off_report.rb

```diff
@@ -0,0 +1,397 @@
+# frozen_string_literal: true
+
+require 'csv'
+begin
+  require 'reporting/cloudwatch_client'
+  require 'reporting/cloudwatch_query_quoting'
+  require 'reporting/command_line_options'
+  require 'reporting/identity_verification_report'
+rescue LoadError => e
+  warn 'could not load paths, try running with "bundle exec rails runner"'
+  raise e
+end
+
+module Reporting
+  class DropOffReport < IdentityVerificationReport
+    include Reporting::CloudwatchQueryQuoting
+
+    module Events
+      IDV_DOC_AUTH_WELCOME = 'IdV: doc auth welcome visited'
+      IDV_DOC_AUTH_WELCOME_SUBMITTED = 'IdV: doc auth welcome submitted'
+      IDV_DOC_AUTH_IMAGE_UPLOAD = 'IdV: doc auth image upload vendor submitted'
+      IDV_DOC_AUTH_CAPTURED = 'IdV: doc auth document_capture visited'
+      IDV_DOC_AUTH_SSN_VISITED = 'IdV: doc auth ssn visited'
+      IDV_DOC_AUTH_VERIFY_VISITED = 'IdV: doc auth verify visited'
+      IDV_DOC_AUTH_VERIFY_SUBMITTED = 'IdV: doc auth verify submitted'
+      IDV_DOC_AUTH_PHONE_VISITED = 'IdV: phone of record visited'
+      IDV_ENTER_PASSWORD_VISITED = 'idv_enter_password_visited'
+      IDV_ENTER_PASSWORD_SUBMITTED = 'idv_enter_password_submitted'
```

Comment:
These events were renamed in #9530, but unfortunately didn't get our`@identity.idp.previous_event_name` tag.

1. Can we go in and add the previous event name tags
2. Can we also query for the previous event names? That will allow this report to be run for dates prior to November 2023

Reply:
yes, i would like to do this! thanks for cluing me into which ones had different names, i wasn't sure how to search for that.